data/src/main/kotlin/com/vultisig/wallet/data/utils/Serializer.kt

```diff
@@ -89,4 +92,36 @@ object KeysignResponseSerializer : KSerializer<tss.KeysignResponse> {
         val surrogate: KeysignResponseSerializable = decoder.decodeSerializableValue(serializer)
         return surrogate.toOriginal()
     }
+}
+
+
+@Singleton
+class CosmosTHORChainResponseSerializer @Inject constructor(
+    private val json: Json,
+) :
+    KSerializer<CosmosTHORChainAccountResponse> {
+    override val descriptor: SerialDescriptor =
+        buildClassSerialDescriptor("SplTokenResponseJsonSerializer")
```

Comment:
why this is `SplTokenResponseJsonSerializer`

Reply:
Fixed